cpp/test/test_distributed_energy_storage.cc

```diff
@@ -0,0 +1,62 @@
+/* Copyright (c) 2016, the Cap authors.
+ *
+ * This file is subject to the Modified BSD License and may not be distributed
+ * without copyright and license information. Please refer to the file LICENSE
+ * for the text and further information on this license.
+ */
+
```

Comment:
what does that test bring that ``test_transient_exact_solution.cc`` doesn't already have?
I mean this has only constant current and that's exactly what the other test does with the gold value isn't it